README.md

```diff
@@ -76,22 +76,23 @@ However far you and your partner get, the application should have, at a minimum,
     -  I can see a list of trips that passenger went on
     - Shows the total amount the passenger has been charged
     - Clicking on an individual trip will bring up details for the trip
-    - On a specific passenger's page I can edit and delete passengers
+    - On a specific passenger's page I can edit and delete that passenger
     - When viewing a passenger's details I can add a new trip
+    	- When creating a new trip, an available driver is selected by the server automatically with no rating
```

Comment:
The wording on this feels a bit odd. I think we should split out the "with no rating" part onto its own line as it doesn't directly relate to the driver selection process.